content/learning-paths/smartphones-and-mobile/afrc/how-to-1.md

```diff
@@ -0,0 +1,32 @@
+---
+title: What is fixed-rate compression
+weight: 2
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## What is fixed-rate compression
+
+[Image compression](https://en.wikipedia.org/wiki/Image_compression) is the process of encoding images so that they can be stored and transmitted using fewer bits than in their original representation.
+This generally results in reduced memory footprint and bandwidth, which helps applications perform better.
+
+[Lossless](https://en.wikipedia.org/wiki/Lossless_compression) compression schemes like [Arm Frame Buffer Compression (AFBC)](https://developer.arm.com/documentation/101897/latest/Buffers-and-textures/AFBC-textures-for-Vulkan?lang=en) achieve this without compromising quality, as they are able to preserve all the original data after decompression, at the expense of certain trade-offs, like reserving memory for compression metadata.
+
+[Arm Fixed Rate Compression (AFRC)](https://developer.arm.com/documentation/101897/latest/Buffers-and-textures/AFRC?lang=en) uses the same bitrate for all compression blocks, so that the encoder and decoder can be well optimized.
+It guarantees a reduced memory footprint and often reduced bandwidth.
+However, this makes the compression [lossy](https://en.wikipedia.org/wiki/Lossy_compression).
+
+AFRC achieves high quality results even with the highest compression ratios, preserving detail in edges and high-frequency areas.
+So much so that, to the naked eye, the compressed and uncompressed images look the same side-by-side, so that it may be called 'visually lossless':
+
+&nbsp;
+| Uncompressed | AFRC (2 bits per component) |
+| ----------- | ----------- |
+| ![Uncompressed](images/sponza_default.png) | ![AFRC 2BPC](images/sponza_afrc.png) |
+
+More information can be found in [this blog](https://community.arm.com/arm-community-blogs/b/graphics-gaming-and-vr-blog/posts/arm-immortalis-g715-developer-overview).
```

Comment:
I think it would be good to add a couple of your graphs (from sample README) showing why AFRC is great here, just before "More information..."
Probably just 1 example each of memory and bandwidth. Have a sentence before the graphs of "AFRC gives significant memory and bandwidth savings:" or similar. Then 2 graphs side-by-side, then "More info can be found..."
Could also link to your sample as well as the blog? Blog is general, but your sample is specific and more detailed? (More info can be found in this blog, and in this vulkan sample.)